### ISSUE TITLE
Use Libevent for socket stuff

### ISSUE THREAD

**Issue by gj12** [CoolerVoid/raptor_waf#9]
Libevent would increase the performance.

**Comment by CoolerVoid** [comment]
yes i talk about it here https://github.com/CoolerVoid/raptor_waf/issues/2

But i need study to implement...is not easy...

**Comment by gj12** [comment]
It's easy for me. I can port it next month.

**Comment by CoolerVoid** [comment]
This is cool, thank you very much!
